lib/ronin/recon/output_formats/archive.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+#
+# ronin-recon - A micro-framework and tool for performing reconnaissance.
+#
+# Copyright (c) 2023-2024 Hal Brodigan (postmodern.mod3@gmail.com)
+#
+# ronin-recon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-recon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with ronin-recon.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/web/spider/archive'
+
+module Ronin
+  module Recon
+    module OutputFormats
+      #
+      # Represents a web archive directory.
+      #
+      class Archive < Ronin::Web::Spider::Archive
```

Comment:
Sub-classing `Ronin::Web::Spider::Archive` might cause problems for us in the future if we ever change it's API. Might be safer to initialize it as an ivar in `initialize`.